aws_lambda_powertools/logging/logger.py

```diff
@@ -129,12 +129,14 @@ def __init__(
         self.sampling_rate = resolve_env_var_choice(
             choice=sampling_rate, env=os.getenv(constants.LOGGER_LOG_SAMPLING_RATE, 0.0)
         )
+        self._is_deduplication_disabled = resolve_truthy_env_var_choice(
```

Comment:
__NIT__: We store whether the deduplication is disabled rather than enabled, then (line 175) we check against this flag not being true. It makes this flag a bit hard to read IMHO. Should we invert this?